compiler/x/env/OMRCPU.cpp

```diff
@@ -27,6 +27,36 @@
 #include "infra/Flags.hpp"
 #include "x/runtime/X86Runtime.hpp"
 
+TR::CPU
+OMR::X86::CPU::detect(OMRPortLibrary * const omrPortLib)
+   {   
+   // Only enable the features that compiler currently uses
+   uint32_t enabledFeatures [] = {OMR_FEATURE_X86_FPU, OMR_FEATURE_X86_CX8, OMR_FEATURE_X86_CMOV,
+                                  OMR_FEATURE_X86_MMX, OMR_FEATURE_X86_SSE, OMR_FEATURE_X86_SSE2,
+                                  OMR_FEATURE_X86_SSSE3, OMR_FEATURE_X86_SSE4_1, OMR_FEATURE_X86_POPCNT,
+                                  OMR_FEATURE_X86_AESNI, OMR_FEATURE_X86_OSXSAVE, OMR_FEATURE_X86_AVX,
+                                  OMR_FEATURE_X86_HLE, OMR_FEATURE_X86_RTM};
+
+   uint32_t featureMasks[OMRPORT_SYSINFO_FEATURES_SIZE];
+   memset(featureMasks, 0, OMRPORT_SYSINFO_FEATURES_SIZE*sizeof(uint32_t));
+   for (size_t i = 0; i < sizeof(enabledFeatures)/sizeof(uint32_t); i++)
+      {
+      uint32_t index = enabledFeatures[i]/32;
+      uint32_t shift = enabledFeatures[i]%32;
+      featureMasks[index] |= (static_cast<uint32_t>(1) << shift);
+      }
```

Comment:
Shouldn't we have an `omrsysinfo_processor_enable_feature` for this?

Reply:
Okay I'll add another `omrsysinfo_processor_enable_feature()` like I added omrsysinfo_processor_disable_feature() for Z.